app/src/ui/multi-commit-operation/multi-commit-operation.tsx

```diff
@@ -14,7 +15,7 @@ export class MultiCommitOperation extends React.Component<
     const { kind } = this.props.state.operationDetail
     switch (kind) {
       case MultiCommitOperationKind.CherryPick:
-        return null
+        return <CherryPick {...this.props} />
```

Comment:
TIL this is possible, and it's something I wondered this weekend and forgot to check, nice! 😆